js/client/modules/@arangodb/testutils/result-processing.js

```diff
@@ -308,24 +314,16 @@ function saveToJunitXML(options, results) {
       }
       let elm = {
         errors: errors,
-        failures: msg,
```

Comment:
This changes renders the local variable `msg` useless. Is that intentional?

Reply:
Good catch! msg can be removed now.